src/gluonts/ext/r_forecast/_predictor.py

```diff
@@ -317,15 +316,20 @@ def predict(
 
         """
         self._warning_message()
-
+        print(f"params is {self.params}")
         for data in dataset:
+
+            if self.method_name == 'arima':
+                target_len = data['target'].shape[0]
+                freq = data['start'].freq
+                self.params = fourier_time_features(target_len, freq, self.prediction_length)(self.params)
```

Comment:
maybe do
```suggestion
               params = fourier_time_features(target_len, freq, self.prediction_length)(self.params)
```
here, so as not to mutate the predictor object